### DISCUSSION TITLE
Sir Can I join your team

### DISCUSSION THREAD

**Discussion by onkarsinghsahil** [ROM-builders/temporary#10615] [General]
respected sir,

I want to join you team
Succeeded build: https://cirrus-ci.com/build/6546117709004800

DT: https://github.com/onkarsinghsahil/android_device_xiaomi_mido_new/commits/xdroid
VT- https://github.com/onkarsinghsahil/proprietary_vendor_xiaomi
KT- https://github.com/onkarsinghsahil/kernel_mido
TG username - @error7951

**Comment by Apon77** [comment]
check #learn in telegram group
